README.md

```diff
@@ -1 +1,41 @@
-# angular-replace-readme
\ No newline at end of file
+# Replacing base markup
+
+## Overview
+
+Another property we can use is `replace` - let's look into that!
+
+## Objectives
+
+- Describe "replace"
+- Replace the base Directive with our template
+
+## What is replace?
+
+When we put our directive into the DOM, you'll notice that what will happen is this:
+
+```html
+<our-directive>
+	<div>
+		Directive contents here!
+	</div>
+</our-directive>
+```
+
+However, you may not want the `our-directive` HTML element to be there.
```

Comment:
Can you add a reason 'why' to this - why is it important/better to replace our directive elements?